scworkercluster.js

```diff
@@ -106,9 +119,15 @@ SCWorkerCluster.prototype._init = function (options) {
   if (options.schedulingPolicy != null) {
     cluster.schedulingPolicy = options.schedulingPolicy;
   }
-  if (options.processTermTimeout) {
+  if (options.processTermTimeout != null) {
```

Comment:
processTermTimeout = 0 wiil return default 10000 since if(0) is false

Reply:
Makes sense :)

Reply:
.